### ISSUE TITLE
Make it work without JS

### ISSUE THREAD

**Issue by aarongustafson** [stowball/color-flashcards#3]
You have a lot of good a11y stuff in there, but you can’t do anything is JavaScript fails for any reason. Can you take another pass and get it working without JavaScript?

**Comment by stowball** [comment]
Yeah, I was wondering about that. The JS is embedded in the page, so it shouldn't fail for network reasons. Is `<noscript>` good enough, or is "enhancing" it on first render required?

Obviously it couldn't be a random game without JS though XD

**Comment by stowball** [comment]
@aarongustafson OK, just pushed up v0.3.1 which has a fallback list of colors if JS fails to initialise. Thanks for the nudge \o/
